tests/unittests/sources/test_vmware.py

```diff
@@ -147,6 +156,18 @@ def test_no_data_access_method(self):
             ret = ds.get_data()
         self.assertFalse(ret)
 
+    def test_convert_to_netifaces_ipv4_format(self):
```

Comment:
Are there any failure or edge cases that should be considered/tested?

Reply:
Added a new test for ipv6 PEER address.